.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
+  - "0.12.9"
```

Comment:
Any thoughts on *0.10*? Is there any reason why not to support it as well? I know it's very old so dropping support may be justifiable.